conf/drupal/config/metatag.metatag_defaults.node__binder.yml

```diff
@@ -5,16 +5,14 @@ dependencies: {  }
 id: node__binder
 label: 'Content: Binder'
 tags:
-  mass_metatag_short_description: '[node:field_binder_short_desc]'
+  mass_metatag_associated_organization: '[node:field_binder_ref_organization]'
   mass_metatag_category: laws-regulations
-  mass_metatag_organization: '[node:mass_organizations]'
+  mass_metatag_contact_details: '[node:field_contact]'
   mass_metatag_date: '[node:field_binder_last_updated:date:metadatadate]'
+  mass_metatag_organization: '[node:mass_organizations]'
+  mass_metatag_resource_type: '[node:field_binder_binder_type:entity:name]'
+  mass_metatag_short_description: '[node:field_binder_short_desc]'
   mg_stakeholder_org: '[node:field_state_organization_tax:entity:name]'
-  mass_metatag_associated_organization: '[node:field_binder_ref_organization]'
-  mass_metatag_type: '[node:field_binder_binder_type:entity:name]'
   mass_metatag_sub_type: '[node:field_binder_binder_type:entity:name]'
-  mass_metatag_contact_details: '[node:field_contact]'
+  mass_metatag_type: '[node:field_binder_binder_type:entity:name]'
   description: '[node:field_binder_short_desc]'
-  og_description: '[node:field_binder_short_desc]'
```

Comment:
I think we should't remove this.

Reply:
Thank you for pointing that out.  It was back on with some other config stripped off.